tests/testthat/test-NetworkMeasures.R

```diff
@@ -0,0 +1,248 @@
+context("Correctness of Network Measures")
+
+##### TEST SET UP #####
+
+rm(list = ls())
+# Configure logger (suppress all logs in testing)
+# expect_silents only work with this logger turned off; only alerts with warnings
+loggerOptions <- futile.logger::logger.options()
+if (!identical(loggerOptions, list())){
+    origLogThreshold <- loggerOptions[[1]][['threshold']]
+} else {
+    origLogThreshold <- futile.logger::INFO
+}
+futile.logger::flog.threshold(0)
+
+##### TESTS #####
+
+###############################
+##### Node Level Measures ##### 
+###############################
+
+# Get Networks
+t <- CreatePackageReport("baseballstats")
+extDepNodes <- t$DependencyReporter$nodes
+intFuncNodes <- t$FunctionReporter$nodes
+
+t2 <- CreatePackageReport("milne"
+                          , pkg_reporters = c(DefaultReporters(), InheritanceReporter$new()
+                                              )
+                          )
+inheritenceNodes <- t2$InheritanceReporter$nodes
+
+expectedNodeLevelMeasures <- c("outDegree"
+                             , "outBetweeness"
+                             , "outCloseness"
+                             , "outSubgraphSize"
+                             , "inSubgraphSize"
+                             , "hubScore"
+                             , "pageRank"
+                             , "inDegree")
+
+extDepNetwork <- t$DependencyReporter$network_measures
+intFuncNetwork <- t$FunctionReporter$network_measures
+inheritenceNetwork <- t2$InheritanceReporter$network_measures
+
+expectedNetworkLevelMeasures <- c("centralization.OutDegree"
+                                  , "centralization.betweenness"
+                                  , "centralization.closeness"
+                                  )
+
+# note, does not test order
+test_that('All expected Node Level Network Measures are calculated', {
+    
+    # External Package Dependencies
+    expect_true(object = all(expectedNodeLevelMeasures %in% names(extDepNodes))
+                , info = 'Network Measures for external package dependencies'
+                )
+    
+    # Internal Function Dependencies
+    expect_true(object = all(expectedNodeLevelMeasures %in% names(intFuncNodes))
```

Comment:
@bburns632 you are the only person I've ever seen use the keyword arg `object = ` in `testthat` code. Is there are reason you like it over this format?

```
expect_true('Network Measures for external package dependencies', {
    all(expectedNodeLevelMeasures %in% names(extDepNodes))
})
```

Not trying to passive-aggressively recommend you change, honestly curious.

Reply:
Just following the docs.  What you wrote above looks like the format for a [test_that()](https://testthat.r-lib.org/reference/test_that.html) call, not an [expect_true()](https://testthat.r-lib.org/reference/logical-expectations.html) call.